src/testbed.cu

```diff
@@ -642,6 +642,13 @@ void Testbed::imgui() {
 			GPUMemory<float> density = get_density_on_grid(res3d, aabb);
 			save_density_grid_to_png(density, fname, res3d, m_mesh.thresh);
 		}
+		ImGui::SameLine();
+		if (ColoredButton("Save RGBA PNGs", 0.2f)) {
+			auto r = get_marching_cubes_res(mc_res);
+			GPUMemory<Eigen::Array4f> rgba = get_rgba_on_grid(r, Eigen::Vector3f(0.0f, 0.0f, 1.0f));
```

Comment:
It'd nicer to pass `view_dir()` rather than z=1. (If the training data doesn't cover all angles well, `view_dir()` is a safer bet for seeing reasonable colors.)